.travis.yml

```diff
@@ -1,11 +1,20 @@
 language: java
 
-install: mvn install -DskipTests=true -Dgpg.skip=true
+before_install:
+  - nvm install $NODE_VERSION
+  - curl -s "https://get.sdkman.io" | bash
+  - source "$HOME/.sdkman/bin/sdkman-init.sh"
+  - sdk install java 17.0.5-amzn
+  - sdk use java 17.0.5-amzn
```

Comment:
Any chance you could provide a short explanation of why you had to make these changes? Why do we have to use sdkman now?

Reply:
For completeness: Travis does not support a java 17 image yet, so we need to get jdk 17 ourselves.